### PR TITLE
Add DaFONT as hard

### PR DESCRIPTION
**Pull request by MattiaRaffaele**
I made some errors and i fixed them _dafont name updated_ is the correct commit

### CODE REVIEW

**Comment by tupaschoal** [comment]
Ok. Do note for the future that you can edit open PRs without closing and opening new ones.

File: _data/sites.json [hunk 1]
```diff
@@ -2983,6 +2983,17 @@
         ]
     },
 
+    {
+        "name": "DaFONT",
+        "url": "https://www.dafont.com/forum/read/12643/how-do-i-delete-my-account",
+        "difficulty": "hard",
+        "notes": "You'll send a private message to Rodolphe : http://www.dafont.com/fr/profile.php?user=1 (Wrote by Vinz).",
+        "notes_it": "Dovrai mandare un messagio privato a Rodolphe http://www.dafont.com/fr/profile.php?user=1 (Scritto da Vinz): [scritto da madrelingua / wrote by a motherlanguage]",
```
Comment (tupaschoal) [inline _data/sites.json:2991]: I'm confused as to whether the note of motherlanguage here is a tip for the user or a note to us, the jdm maintainers. Can you clarify your intent, please?

File: _data/sites.json [hunk 2]
```diff
@@ -2983,6 +2983,17 @@
         ]
     },
 
+    {
+        "name": "DaFONT",
+        "url": "https://www.dafont.com/forum/read/12643/how-do-i-delete-my-account",
+        "difficulty": "hard",
+        "notes": "You'll send a private message to Rodolphe : http://www.dafont.com/fr/profile.php?user=1 (Wrote by Vinz).",
```
Comment (tupaschoal) [inline _data/sites.json:2990]: What should this "Wrote by Vinz" mean?

Comment (MattiaRaffaele) [reply _data/sites.json:2990]: Vinz (dafont user) wrote "You'll send a private message to Rodolphe" in dafont forum here: http://www.dafont.com/fr/profile.php?user=1

File: _data/sites.json (hunk 1 above)
Comment (MattiaRaffaele) [reply _data/sites.json:2991]: I wrote this to let the italian justdeleteme users that the translation is 100% accurate

**Comment by MattiaRaffaele** [comment]
> 

Thank you very much!
Regards

File: _data/sites.json (hunk 2 above)
Comment (tupaschoal) [reply _data/sites.json:2990]: I see, in this case I don't think we need to add this information and can be a bit more succint.
```suggestion
        "notes": "Send a private message to Rodolphe: <a href='http://www.dafont.com/fr/profile.php?user=1'>.",
```

File: _data/sites.json (hunk 1 above)
Comment (tupaschoal) [reply _data/sites.json:2991]: I see, there's no need for this note, but thanks for letting us know.
```suggestion
        "notes_it": "Dovrai mandare un messagio privato a Rodolphe: <a href='http://www.dafont.com/fr/profile.php?user=1'>",
```

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**